src/components/DateTimePicker/DateTimePicker.js

```diff
@@ -0,0 +1,137 @@
+import React, { useState, useEffect } from "react";
+import throttle from "lodash.throttle";
+import { useTranslation } from "react-i18next";
+import Picker from "react-datepicker";
+
+import "react-datepicker/dist/react-datepicker.css";
+import CustomInput from './CustomInput';
+
+import en from "date-fns/locale/en-US";
+import ru from "date-fns/locale/ru";
+import hy from "date-fns/locale/hy";
+
+import {
+  DATE_FORMATS,
+  TIME_FORMATS,
+  DATE_TIME_FORMATS
+} from "../../config"
+
+
+const DateTimePicker = props => {
+
+  const LOCALES = { en, ru, hy };
+  const FORMATS = { 
+    date: DATE_FORMATS,
+    time: TIME_FORMATS,
+    datetime: DATE_TIME_FORMATS
+  }
+  const BREAKPOINT = 1000; // width less than this is considered small
```

Comment:
We should give this a better thought, do we want small to be displayed on mobile and tablet? if so we need to sync this with our media query numbers in CSS. @ao-armen which is a starting point for our desktop. Basically of how many px width we consider application top be in the desktop mode?

Reply:
I moved this constant into the config as well